example/mocks/delete/DELETE.mock

```diff
@@ -0,0 +1,5 @@
+#import '../response_status.js';
+content-type: application/json; charset=utf-8
+server: mock-server
+
+#import '../response_body.js';
```

Comment:
nit: might be good to add end of line (same for others)

Reply:
done